### PR TITLE
Prototype routing

### PR DESCRIPTION
**Pull request by lin8231**
* App now routes properly between login and the sgonks platform 
* Dashboard's buttons are now working 
* Made styling amends to buttons since styling was broken when button was replaced with Link tag

### CODE REVIEW

**Comment by roonil-wazlib** [comment]
That bot isn't going to be happy with the JS formatting until you completely screw it up by the looks of things.

File: makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CLANG_FORMAT=node_modules/clang-format/bin/linux_x64/clang-format --style=Google
+CLANG_FORMAT=node_modules/clang-format/bin/darwin_x64/clang-format --style=Google
```
Comment (roonil-wazlib) [inline makefile:1]: We need to change the Makefile to check OS but this is ok for now I guess.

File: portfolio/frontend/src/containers/SGonksPlatform/Dashboard/Dashboard.module.css [hunk 2]
```diff
@@ -6,13 +6,20 @@
 
 .Box {
     padding: 10px;
-    border-style: solid;
+    border: solid 1px lightgray;
     margin-top: 10px;
     margin-bottom: 10px;
+    border-radius: 7px;
+    flex: 0 0 30%;
+
```
Comment (roonil-wazlib) [inline portfolio/frontend/src/containers/SGonksPlatform/Dashboard/Dashboard.module.css:14]: Remove this line?

Comment (lin8231) [reply portfolio/frontend/src/containers/SGonksPlatform/Dashboard/Dashboard.module.css:14]: yass

**Review by roonil-wazlib — APPROVED** [review]

File: makefile (hunk 1 above)
Comment (roonil-wazlib) [reply makefile:1]: Fixed the makefile, so this should be overwritten on merge.

File: portfolio/frontend/src/containers/Login/Login.js [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
         <div>
             <div className={classes.Header}>
                 <img src="sGonksLogo.png" alt="sGonks Logo" className={classes.Logo}></img>
-                <div className={classes.HeaderButtons}>
-                    <button className={classes.Login} onClick={() => props.handler()}>Login</button>
-                    <button className={classes.Help}>See how it works</button>
+                <div className={classes.Buttons}>
+                    <Link className={classes.Login} to="/sgonks-platform">Login</Link>
+                    <Link className={classes.Help} to="/explanation">See how it works</Link>
                 </div>
```
Comment (leonnemets) [inline portfolio/frontend/src/containers/Login/Login.js:13]: Can this be factored out with the buttons on line 24 and 25

File: portfolio/frontend/src/containers/Login/Login.module.css [hunk 4]
```diff
@@ -41,15 +41,26 @@ p {
 
 .Login {
     width: 200px;
-    background-color: blue;
     color: white;
     margin-right: 10px;
+    padding-top: 10px;
+    height: 30px;
```
Comment (leonnemets) [inline portfolio/frontend/src/containers/Login/Login.module.css:47]: Buttons have lots of common properties/attributes, can this be factored out

File: portfolio/frontend/src/containers/Login/Login.js (hunk 3 above)
Comment (lin8231) [reply portfolio/frontend/src/containers/Login/Login.js:13]: I'm not sure what you mean. Could you clarify?

Comment (leonnemets) [reply portfolio/frontend/src/containers/Login/Login.js:13]: Factor out common code to function

Comment (roonil-wazlib) [reply portfolio/frontend/src/containers/Login/Login.js:13]: Can you do that with html?

File: portfolio/frontend/src/containers/Login/Login.module.css (hunk 4 above)
Comment (lin8231) [reply portfolio/frontend/src/containers/Login/Login.module.css:47]: I don't believe you can do this in HTML, but will make a fix for overlapping CSS properties.

Comment (lin8231) [reply portfolio/frontend/src/containers/Login/Login.module.css:47]: Done. Please re-review :)

**Review by leonnemets — APPROVED** [review]

**Pull request merged**